h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/XGBoostUtils.java

```diff
@@ -94,7 +94,7 @@ public static DMatrix convertFrameToDMatrix(Key<DataInfo> dataInfoKey,
         }
 
         if (sparse) {
-            Log.info("Treating matrix as sparse.");
+            Log.debug("Treating matrix as sparse.");
```

Comment:
There was a time when we explicitely wanted this to be INFO. I agree with the change. Just FYI.

Reply:
Yep, it was filling the logs with these messages - they don't have a meaning to the user so I disabled it.